### ISSUE TITLE
Markers don't show.

### ISSUE THREAD

**Issue by aghasidoor3** [IbrahimTabba/custom_map_markers#4]
On real device, when i create apk or via testflight on Ios device,show CircularProgressIndicator,don't show map!

**Comment by LogicNg** [comment]
It won't work for animation.

**Comment by FaisalMohammadi** [comment]
yes i have the same problem. did anybody found a solution?

**Comment by PisaIlogs** [comment]
The markers in my iOS app are shown properly at first, but when I move the app to the background and after some time to the foreground again the markers are gone.

**Comment by FaisalMohammadi** [comment]
yah for me as well, when i debug the app everything is fine when i install the app in release mode the same problem happens like the pins arte not visible at the biginning. Is their any update to this.

**Comment by PisaIlogs** [comment]
I am currently using @KishanBusa8's branch, which solves the issue for me. Hopefully it will be merged into the main branch soon.

**Comment by FaisalMohammadi** [comment]
yes i can confirm this as well. Please merge the pull request @IbrahimTabba
